controllers/common/const.go

```diff
@@ -49,6 +49,9 @@ const (
 	CMName      = "meridio-configuration"
 
 	NetworkServiceName = "external-vlan"
+
+	ResourceRequirementKey          = "resourcerequirement"
```

Comment:
@johlj or @tedlean any suggestion?

Reply:
This is meant to refer to a template that is to inserted as "resources" section in the container manifest, right?
So my suggestion would be "resources-template" or "resource-template"